tmk_core/protocol/vusb/vusb.c

```diff
@@ -112,31 +112,25 @@ static void send_mouse(report_mouse_t *report) {
     }
 }
 
-typedef struct {
-    uint8_t  report_id;
-    uint16_t usage;
-} __attribute__((packed)) report_extra_t;
-
-static void send_system(uint16_t data) {
+static void send_extra(uint8_t report_id, uint16_t data) {
+    static uint8_t last_id    = 0;
     static uint16_t last_data = 0;
-    if (data == last_data) return;
+    if ((report_id == last_id) && (data == last_data)) return;
```

Comment:
Before there was a separate last_data for system and consumer, now its shared between the two. If called like
```c
send_system(X);
send_consumer(Y);
send_system(X);
```
send_system will be sent again, when previously it would not have been.

Will this cause any potential issues?

Reply:
No, in fact IMO that's actually a bug, as the system and consumer usages get sent over the same interface, so it makes more sense to pull it up into `send_extra()`.

Reply:
I think also, it doesn't really matter, as `last_data` should be getting set back to `0` on keyup.